applications/crossbar/src/crossbar_util.erl

```diff
@@ -956,33 +956,53 @@ load_descendants_count(ViewOptions) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec maybe_update_descendants_count(ne_binary(), integer()) -> 'ok'.
+-spec maybe_update_descendants_count(ne_binary(), integer(), integer()) -> 'ok'.
+-spec maybe_update_descendants_count(ne_binary(), wh_json:object(), integer(), integer()) -> 'ok'.
+-spec maybe_update_descendants_count(ne_binary(), wh_json:object(), integer(), integer(), integer()) -> 'ok'.
+
 maybe_update_descendants_count(AccountId, NewCount) ->
+    maybe_update_descendants_count(AccountId, NewCount, 3).
+
+maybe_update_descendants_count(AccountId, _, Try) when Try =< 0 ->
+    io:fromat("too many attempts to update descendants count for ~s~p", [AccountId]);
```

Comment:
This function does not exist. Also, why no lager? Or, if you do need to use io, you need the `~n` to add an endline.